coralproject/talk
# [CORL-2662]: fix to show comment box for QA mode in alternate oldest first sort

## What does this PR do?

These changes update to address the issue where, in Q&A story mode, in the alternate oldest sort view, the comment box was not shown. Now, it is shown as expected beneath the initial `Answered` comments in that first tab.

In addition, it updates the `Add a comment/question` button scrolling to also work as expected for the featured/answered comments tabs. Now, that button knows which tab the user is on and scrolls accordingly (with Virtuoso if on all questions/comments, and without if it's on the featured/answered comments tab).

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Go to a Q&A story. Sort by oldest first. Have `ALTERNATE_OLDEST_FIRST_VIEW` feature flag enabled. Have at least 1 answered question and go to the `Answered` tab. See that the comment box is displayed below all answered comments. Click on the `Ask a question` button. See that you are scrolled down to the box. Have enough answered questions that you need to be scrolled down the page to it, and confirm that it's working as expected. Navigate to the `All` questions tab and confirm that the `Ask a question` button and comment box are both there and the button is still scrolling you down the page to it as expected.

Go to a comments mode story. Test all of the same things to confirm that it is all still working as expected, too.
 
## How do we deploy this PR?
